client/assets/helpers.js

```diff
@@ -0,0 +1,289 @@
+let editData
+
+//page
+function homepage () {
+    $('#list').hide()
+    $('#signin').hide()
+    $('#signup').hide()
+    $('#logout').hide()
+}
+
+function listpage () {
+    $('#list').show()
+    $('#edittodo').hide()
+    $('#home').hide()
+    $('#signin').hide()
+    $('#signup').hide()
+    $('#navSignup').hide()
+    $('#navLogin').hide()
+    fetchTodo()
+    weather()
+    videos()
+}
+
+function loginpage() {
+    $('#signin').show()
+    $('#edittodo').hide()
+    $('#home').hide()
+    $('#list').hide()
+    $('#signup').hide()
+}
+
+function signuppage() {
+    $('#signup').show()
+    $('#home').hide()
+    $('#list').hide()
+    $('#signin').hide()
+}
+
+function editpage(id) {
+    $('#list').show()
+    $('#edittodo').show()
+    $('#createtodo').hide()
+    $('#home').hide()
+    $('#signin').hide()
+    $('#signup').hide()
+    $('#navSignup').hide()
+    $('#navLogin').hide()
+    fetchTodoByid(id)
+}
+
+//function
+function signup() {
+    let email = $('#emailSignup').val()
+    let password = $('#passwordSignup').val()
+    console.log(email, password);
+    $.ajax({
+        method: 'POST',
+        url: 'http://localhost:3000/register', 
+        data: {email: email, password: password}
+    })
+    .done(data => {
+        console.log(data)
+    })
+    .fail(err => {
+        console.log(err)
+    })
+    .always(() => {
+        $('#registerForm').trigger('reset')
+    })
+}
+
+function login() {
+    let email = $('#emailSignup').val()
```

Comment:
ini select input nya salah malah ke #emailSignUp harusnya ke yg sign in makanya login nya ga dapet datanya, jangan dibiasakan copas ya